Ladybird/Android/src/main/java/org/serenityos/ladybird/WebView.kt

```diff
@@ -0,0 +1,29 @@
+package org.serenityos.ladybird
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.util.AttributeSet
+import android.view.View
+
+class WebView(context: Context, attributeSet: AttributeSet) : View(context, attributeSet) {
```

Comment:
This should (eventually) implement `NestedScrollingChild3` and `ScrollingView`

Reply:
Looks reasonable, I'll drop a FIXME. Looks like doing that requires help from the view implementation.